utils/model_uploader/model_listing/pretrained_models_all_versions.json

```diff
@@ -0,0 +1,173 @@
+[
+  {
+    "name": "huggingface/sentence-transformers/all-MiniLM-L12-v2",
+    "versions": {
```

Comment:
Is this not going to affect the pre-training model upload. Found an issue on AOS 2.9 with such format

Reply:
We already fix the workflow and have it create a new file instead of overwrite the old model listing file.

Reply:
We will then make changes to ml-commons so that it relies on this new format in the future release.